packages/fxa-auth-server/lib/luaScripts/removeAccessToken_1.lua

```diff
@@ -1,8 +1,6 @@
-local tokenId = KEYS[1]
-local _, _, prefix = tokenId:find('(%a+:)')
```

Comment:
this prefix logic was fragile and since `getAccessTokens` prunes missing ids from the index it's better to just let the index get updated there.